src/sensitivity.jl

```diff
@@ -54,3 +54,28 @@ function plotUnitSensitivity(paramRange, result, realParam, i)
     plot!([realParam], seriestype = "vline", margin = 0.3cm, legend = :false)
     ylims!((1E2, 1E5))
 end
+
+""" This function calculates the simulated cell number for a pair of drugs in their exact concentration.  """
+function calc_cellNum(pDr1, pDr2, g0)
+    Dr1CellNum = predict(pDr1, g0, 96.0)
+    Dr2CellNum = predict(pDr2, g0, 96.0)
+    normNum = 1.0 .- [Dr1CellNum/g0, Dr2CellNum/g0]
+    combin = -((normNum[1] + normNum[2] - normNum[1] * normNum[2]) .- 1.0) * g0 
+    return combin
+end
+
+""" Calculates the difference between the bliss_cell number and bliss_params. """
+function calc_diff(Dr1_params, Dr2_params, effs, combin, concs, conc1_indx, conc2_indx, g0)
+    bliss_comb = DrugResponseModel.fullCombinationParam(Dr1_params, Dr2_params, effs, 8, conc1_indx, conc2_indx)
+    bliss_comb_cellnum = BlissModelComb(bliss_comb, g0)[conc1_indx, conc2_indx]
+    return combin - bliss_comb_cellnum
+end
+
+fd(x) = calc_diff(x, Dr2_params, effs, combin, concs, conc1_indx, conc2_indx, g0)
+
+function get_derivative(x, combin, concs, Dr1_indx, Dr2_indx, k)
+    # "k" is the index of the parameter we want to calculate the derivative of diff with respect to it.
+    fd(x) = calc_diff(x, combin, concs, Dr1_indx, Dr2_indx) # closure?
+    out = DiffResults.DiffRresult(x[k])
+    return ForwardDiff.derivative(out, fd, x[k])
+end
```

Comment:
I have difficulty handling the differentiation. I couldn't find an example online that could help with the correct syntax.
I know that I should use closure to set the input of the `calc_diff` function only the parameters, that's why I defined `fd(x)` but I think clearly it is wrong. 
All the examples I saw were written without any parameter and they had an input vector where they were differentiating the function w.r.t. the input vector. I don't know how to tell the function to take the derivative of this function only with respect to one of the elements in the parameter vector. Maybe it is not necessary...